modules/extensions/cilium.tf

```diff
@@ -2,45 +2,47 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
 locals {
-  cilium_helm_crds_file       = join("/", [local.yaml_manifest_path, "cilium.crds.yaml"])
-  cilium_helm_manifest_file   = join("/", [local.yaml_manifest_path, "cilium.manifest.yaml"])
-  cilium_helm_values_file     = join("/", [local.yaml_manifest_path, "cilium.values.yaml"])
-  cilium_net_attach_def_file  = join("/", [local.yaml_manifest_path, "cilium.net_attach_def.yaml"])
-  cilium_veth_config_map_file = join("/", [local.yaml_manifest_path, "cilium.cni_config_map.yaml"])
-
-  cilium_helm_crds     = one(data.helm_template.cilium[*].crds)
-  cilium_helm_manifest = one(data.helm_template.cilium[*].manifest)
+  cilium_helm_crds_file            = join("/", [local.yaml_manifest_path, "cilium.crds.yaml"])
+  cilium_helm_manifest_file        = join("/", [local.yaml_manifest_path, "cilium.manifest.yaml"])
+  cilium_helm_values_file          = join("/", [local.yaml_manifest_path, "cilium.values.yaml"])
+  cilium_helm_values_override_file = join("/", [local.yaml_manifest_path, "cilium.values-override.yaml"])
+  cilium_net_attach_def_file       = join("/", [local.yaml_manifest_path, "cilium.net_attach_def.yaml"])
+  cilium_veth_config_map_file      = join("/", [local.yaml_manifest_path, "cilium.cni_config_map.yaml"])
+
+  cilium_helm_crds            = one(data.helm_template.cilium[*].crds)
+  cilium_helm_values_override = one(data.helm_template.cilium[*].values)
+  
+  cilium_helm_repository      = "https://helm.cilium.io"
 
   cilium_vxlan_cni = {
     install      = true
-    chainingMode = "none"
     exclusive    = true # !var.multus_install
   }
 
-  # TODO Support Flannel w/ generic-veth & tunnel disabled
-  cilium_tunnel = "vxlan" # var.cni_type == "flannel" ? "disabled" : "vxlan"
-  cilium_flannel_cni = {
-    install      = true
-    chainingMode = "generic-veth"
-    configMap    = "cni-configuration"
-    customConf   = var.cni_type == "flannel"
-    exclusive    = !var.multus_install
-  }
-
   cilium_helm_values = {
     annotateK8sNode                 = true
-    cluster                         = { name = "oke-${var.state_id}" }
-    clustermesh                     = { useAPIServer = true }
+    cluster                         = {
+      name = "oke-${var.state_id}"
+      id   = 1
+    }
+    clustermesh                     = { 
+      useAPIServer = false 
+      apiserver = {
+        kvstoremesh = {
+          enabled = false
+        }
+      }
+    }
     cni                             = local.cilium_vxlan_cni
-    containerRuntime                = { integration = "crio" }
-    installIptablesRules            = true
     installNoConntrackIptablesRules = false
     ipam                            = { mode = "kubernetes" }
```

Comment:
same thing for ipam mode. set the default to "kubernetes" though

Reply:
can be overridden using the `cilium_helm_values` variable.